devDocs/startupShutdown.md

```diff
@@ -43,41 +57,35 @@
     1. All wx windows are closed
     1. Now that windows are closed, a new NVDA instance is started if requested
 
-### Out of process: `WM_QUIT`
-* [a windows message](https://docs.microsoft.com/en-us/windows/win32/winmsg/wm-quit) which can be sent across processes which will force the main loop to exit and close most NVDA windows.
+### When exiting from `WM_QUIT`
+* [A Windows Message](https://docs.microsoft.com/en-us/windows/win32/winmsg/wm-quit) received from an external process, such another NVDA process.
+* NVDA accepts [WM_QUIT](#When-exiting-from-WM_QUIT) messages from other processes and creates a [named window](https://docs.microsoft.com/en-us/windows/win32/learnwin32/creating-a-window#creating-the-window) that can be discovered.
+* Will force the main loop to exit and close most wx Windows.
 * We subsequently run `triggerNVDAExit` to ensure that clean up code isn't missed, and pump the queue to execute it.
 * Changing to using a custom message, which would allow a custom handling (eg just `triggerNVDAExit`), requires compatibility with messaging older NVDA versions that are only aware of `WM_QUIT`.

```

Comment:
```suggestion
* Using a custom message has been considered:
  - Would allow custom handling (eg just `triggerNVDAExit`)
  - Unfortunately, older NVDA versions will only be aware of `WM_QUIT`, so we'd need to send `WM_QUIT` to these versions.
  - Sending the custom message, waiting for a timeout, then sending `WM_QUIT` was tried but....?
  - Identifying the running version (to selectively send the message) was unsuccessful because ...?

```

This point about using a custom message instead of WM_QUIT could probably be under its own heading. There is plenty to discuss about the things tried and the obstacles.

Reply:
I think work here was mostly limited due to it being unnecessary and adding scope. I've added what notes I could:

```suggestion
* Using a custom message has been considered:
  - Would allow custom handling (eg just `triggerNVDAExit`)
  - Unfortunately, older NVDA versions will only be aware of `WM_QUIT`, so we'd need to send `WM_QUIT` to these versions.
  - Sending the custom message, waiting for a timeout, then sending `WM_QUIT` adds a significant wait time
  - Identifying the running version (to selectively send the message) requires maintaining 2 message windows in NVDA (one for legacy behaviour) and adds complexity
```